parm/config/gfs/config.prepoceanobs

```diff
@@ -15,6 +16,10 @@ export OBS_YAML=${OBS_LIST}
 # ocean analysis needs own dmpdir until standard dmpdir has full ocean obs
 export DMPDIR=@DMPDIR@
 
+# For BUFR2IODA json and python scripts
+export JSON_TMPL_DIR=${HOMEgfs}/sorc/gdas.cd/parm/ioda/bufr2ioda
+export BUFR2IODA_PY_DIR=${HOMEgfs}/sorc/gdas.cd/ush/ioda/bufr2ioda
```

Comment:
This needs to be linked to `HOMEgfs` and then use `USHgfs` instead of referencing code from `sorc/gdas.cd/ush`.

Reply:
done